### PR TITLE
🔎 Added browse pages to robots.txt

### LINKED ISSUE #2599: 📄 Redirect/Archive/Migrate BrowseXML.aspx
cc: @bahjatalaadelssw @tiagov8 

URL: https://www.ssw.com.au/ssw/BrowseXML.aspx/Browse.aspx

As per my conversation with @amankumarrr yesterday, Bahjat found this page from a Google search (see his email below). 

1. Please decide what to do with this page (Redirect/Archive/Migrate/other?).

---

Email from Bahjat:

Hi SysAdmins,

As per my conversation with Kiki, a Google search of “SSW standards” resulted in this link
              https://www.ssw.com.au/ssw/BrowseXML.aspx/8Steps/8Steps/8Steps/Access2007/AccessReporter/Advertising/AgileTemplate/Angular/App_Readme/App_Start/App_Themes/blog/Bot/browse.aspx

Image
Figure: Search results

Image
Figure: XML site for SSW

### PR DESCRIPTION
**Pull request by Calinator444**
### Description 
As mentioned in the linked issue, Google is attempting to index a series of pages that weren't ported resulting in errors.

These pages were not ported because they are generated and do not contain unique content (There's a bug on the original V1 server causing these pages to be generated recursively so there's around 22,000 of these based on the google search console results).

This PR adds wildcard routes to robots.txt too ensure google doesn't attempt to index them.

https://searchengineland.com/a-deeper-look-at-robotstxt-17573

- Affected routes: /robots.txt.

- Fixed #2599

- [x] Include done video or screenshots

image

**Figure**: **Updated robots.txt file**

### CODE REVIEW

File: next-sitemap.config.js
Comment (jeoffreyfischer) [inline next-sitemap.config.js]: Please add the following too:
/* browsexml.aspx* (I intentionally added a space due to markdown display - remove the space when you implement it)
/*browsexml.aspx$

File: next-sitemap.config.js
Comment (jeoffreyfischer) [inline next-sitemap.config.js]: Check if the other URLs in the list would also need * and/or $

**Review by jeoffreyfischer — CHANGES_REQUESTED** [review]

File: next-sitemap.config.js
Comment (Calinator444) [reply next-sitemap.config.js]: Done - I added both of these routes to robots.txt

File: next-sitemap.config.js
Comment (Calinator444) [reply next-sitemap.config.js]: For these 2 routes in particular I am using the `$` because I only want to exclude routes ending with `browse.aspx`


According to the app insights logs were not getting any requests that *contain* `browse.aspx` unless they *end with* it


image

Comment (Calinator444) [reply next-sitemap.config.js]: I did remove 
These routes because the wildcard routes already cover both edge cases
`/*BrowseXML.aspx$`
`/*browsexml.aspx$`

File: next-sitemap.config.js
Comment (jeoffreyfischer) [inline next-sitemap.config.js]: I would also use the * instead of $ on these 2 lines just in case

**Review by jeoffreyfischer — CHANGES_REQUESTED** [review]

**Comment by jeoffreyfischer** [comment]
Before merging, can you check if that's working on the latest slot?

File: next-sitemap.config.js
Comment (Calinator444) [reply next-sitemap.config.js]: Done - I modified these routes to be wildcard routes

**Comment by Calinator444** [comment]
@jeoffreyfischer 

> Before merging, can you check if that's working on the latest slot?

Done - here's how the robots.txt file looks on the latest staging slot. It looks like were good to go.


image

**Comment by Calinator444** [comment]
@jeoffreyfischer does this need any more changes?

blocked/need your approval

thanks

**Review by jeoffreyfischer — APPROVED** [review]

**Comment by jeoffreyfischer** [comment]
LGTM

**Pull request merged**
